.github/workflows/check.yaml

```diff
@@ -8,7 +8,7 @@ on:
       - "v[0-9]*.[0-9]*.x"
 
 jobs:
-  Setup Environment:
+  Setup_Environment:
```

Comment:
Are step names not allowed to have spaces?

Reply:
unfortunately seems like they are not: https://github.com/citizennet/purescript-httpure/actions/runs/2272722569

> The workflow is not valid. .github/workflows/check.yaml (Line: 11, Col: 3): The identifier 'Setup Environment' is invalid. IDs may only contain alphanumeric characters, '_', and '-'. IDs must start with a letter or '_' and and must be less than 100 characters. .github/workflows/check.yaml (Line: 62, Col: 3): The identifier 'Validate Purescript Format' is invalid. IDs may only contain alphanumeric characters, '_', and '-'. IDs must start with a letter or '_' and and must be less than 100 characters.

Reply:
Ah gotcha, thanks for the explanation!